scripts/docs/generate_docc.sh

```diff
@@ -35,48 +35,13 @@ modules=(
   DistributedActors
 )
 
-declare -r SWIFT="$TOOLCHAIN/usr/bin/swift"
-
-# all our public modules which we want to document, begin with `DistributedActors`
```

Comment:
This a duplication to the above line

Reply:
Should we add "DistributedActorsTestKit" to sync with the "modules" definition in preview_docc.sh

Reply:
Yeah, I think you're right -- we want to document the testkit too 👍